_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "Online Workshop Recipe"
+---
+
+# A Recipe for an Online Carpentries Workshop
+
+In the past year+ of teaching online, our community has learned many things. Our initial [online training recommendations](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop) recognized that we were in uncharted territory for Carpentries workshops! In consulting with community members, we heard many different narratives and suggestions. Rather than creating a single 'recipe,' we chose to distill that advice into 'things to think about', giving Instructors space to engineer their own solutions to the challenges presented by teaching online.
```

Comment:
Just a note that we replace z in words like recognized with s - recognised

Reply:
```suggestion
In the past year+ of teaching online, our community has learned many things. Our initial [online training recommendations](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop) recognised that we were in uncharted territory for Carpentries workshops! In consulting with community members, we heard many different narratives and suggestions. Rather than creating a single 'recipe,' we chose to distill that advice into 'things to think about', giving Instructors space to engineer their own solutions to the challenges presented by teaching online.
```